samples/local_server_tls_sample/tls_server_io_schannel.c

```diff
@@ -464,6 +464,14 @@ static int internal_send(TLS_IO_INSTANCE* tls_io_instance, const void* buffer, s
     return result;
 }
 
+
+// A simple sample callback.
+static void sample_on_send_complete(void* context, IO_SEND_RESULT send_result)
```

Comment:
>sample_ [](start = 12, length = 7)

Can you please remove the sample prefix as this might move to src at some point. I'd rather have it unckecked_

Reply:
"I'd rather have it unckecked_"... Will do.